samples/src/main/java/com/example/bigquery/GetTable.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.bigquery;
+
+// [START bigquery_get_table]
+import com.google.cloud.bigquery.BigQuery;
+import com.google.cloud.bigquery.BigQueryException;
+import com.google.cloud.bigquery.BigQueryOptions;
+import com.google.cloud.bigquery.Table;
+import com.google.cloud.bigquery.TableId;
+
+public class GetTable {
+
+  public static void runGetTable() {
+    // TODO(developer): Replace these variables before running the sample.
+    String projectId = "bigquery_public_data";
+    String datasetName = "samples";
+    String tableName = "shakespeare";
+    getTable(projectId, datasetName, tableName);
+  }
+
+  public static void getTable(String projectId, String datasetName, String tableName) {
+    try {
+      // Initialize client that will be used to send requests. This client only needs to be created
+      // once, and can be reused for multiple requests.
+      BigQuery bigquery = BigQueryOptions.getDefaultInstance().getService();
+      TableId tableId = TableId.of(projectId, datasetName, tableName);
+      Table table = bigquery.getTable(tableId);
+      System.out.println("Table info: " + table.getDescription());
+      System.out.println("Table retrieved successfully.");
```

Comment:
nit: This line seem redundant - It was already successful if you see `"Table info"`